cogs/basic_command.py

```diff
@@ -22,15 +22,17 @@ async def bothelp(self, ctx):
                 'can be one of \'all\', \'troop\', \'spell\' or \'building\'. Defaults to \'all\' if not specified',
                 f'[!ausclanboat]({url}) - Lists clan members who attacked enemy boats this week.',
                 f'[!ausclanwar [weeks]]({url}) - Shows the war peformance of the players in x amount of weeks. Defaults to 4 weeks if not specified',
-                f'[!ausclanremaining]({url}) - Shows the remaining war attacks & remaining players count for the current river race'
+                f'[!ausclanremaining]({url}) - Shows the remaining war attacks & remaining players count for the current river race',
+                f'[!ausclanwarplayers]({url}) - Shows all players within the clan and players who left the clan who has not completed all war attacks'
```

Comment:
Wouldn't a better name be "ausclanremainingplayers", since.... you are finding the `remaining` players who haven't done their war. 

Likewise how `ausclanremaining` finds the `remaining` war attacks count

Reply:
done